flink-runtime-web/src/test/java/org/apache/flink/runtime/webmonitor/handlers/JarHandlers.java

```diff
@@ -120,8 +120,10 @@ public static String uploadJar(
     public static JarListInfo listJars(JarListHandler handler, RestfulGateway restfulGateway)
             throws Exception {
         HandlerRequest<EmptyRequestBody, EmptyMessageParameters> listRequest =
-                new HandlerRequest<>(
-                        EmptyRequestBody.getInstance(), EmptyMessageParameters.getInstance());
+                HandlerRequest.create(
+                        EmptyRequestBody.getInstance(),
+                        EmptyMessageParameters.getInstance(),
+                        Collections.emptyList());
```

Comment:
It would be nice to have some factory methods with fewer parameters, so we have to pass around less empty lists/maps. Empty body and parameters feel fine, but the meaning of this empty list is not clear at first glance.
At least for the produciton code occurrences...

Reply:
There's only a single production code occurrence in `AbstractHandler` though. All other usages are for tests.

I'm fine with adding a `create` variant without a `uploadedFiles` arguments. Most of the test usages of `resolveParametersAndCreate` should eventually be migrated to `create` for simplicity. I didn't do that in this PR to keep the scope a bit smaller.